lib/Github/Exception/ErrorException.php

```diff
@@ -7,6 +7,6 @@
  *
  * @author Joseph Bielawski <stloyd@gmail.com>
  */
-class ErrorException extends \ErrorException implements ExceptionInterface
+class ErrorException extends \Exception implements ExceptionInterface
```

Comment:
Why is this change needed?

Reply:
Not sure anymore, it's quite a long time.

I think it was related to mocks in some way. Try running test with/without it please to confirm.